release/src/main/scripts/build_release_candidate.sh

```diff
@@ -280,15 +280,12 @@ echo "2. Source distribution deployed to https://dist.apache.org/repos/dist/dev/
 echo "3. Website pull request published the Java API reference manual the Python API reference manual."
 
 echo "==============Things Needed To Be Done Manually=============="
-echo "1.You need to update website updates PR with a new commit: "
+echo "1.Make sure a pull request is created to update the javadoc and pydoc to the beam-site: "
 echo "  - cd ~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_WEBSITE_REPO}/${WEBSITE_ROOT_DIR}"
 echo "  - git checkout updates_release_${RELEASE}"
-echo "  - Add new release into src/get-started/downloads.md "
+echo "  - Check if both javadoc/ and pydoc/ exist."
 echo "  - commit your changes"
-echo "2.You need to update website updates PR with another commit: src/get-started/downloads.md"
-echo "  - add new release download links like commit: "
-echo "    https://github.com/apache/beam-site/commit/29394625ce54f0c5584c3db730d3eb6bf365a80c#diff-abdcc989e94369c2324cf64b66659eda"
-echo "  - update last release download links from release to archive like commit: "
-echo "    https://github.com/apache/beam-site/commit/6b9bdb31324d5c0250a79224507da0ea7ae8ccbf#diff-abdcc989e94369c2324cf64b66659eda"
+echo "2.Create a pull request to update the release in the beam/website:"
+echo "  - An example pull request：https://github.com/apache/beam/pull/9341"
```

Comment:
The `Release Note` url may not very obvious to get. We have an instruction [here](https://beam.apache.org/contribute/release-guide/#review-release-notes-in-jira). Can you add this guide here in case people don't know how to find that?

Reply:
Added a link of the release note.